recipe/meta.yaml

```diff
@@ -1,18 +1,30 @@
-{% set version = "8.1.0.77" %}
+{% set version = "8.2.0.53" %}
+
+# cuDNN v8.2.0 supports CUDA versions 10.2 - 11.3
+# For CUDA 10.2 there is a separate cuDNN distribution and a link for download
+# For CUDA 11.x there is one cuDNN distribution that works with all CUDA 11 versions
+
+# This Jinja code below causes early expansion of cuda_compiler_version, leading to
+# unexpected zippping with the key "cudnn" in the .ci_support files...
+# {% if cuda_compiler_version[:2] == "11" %}
+# {% set cudnn_cuda_version = "11.3" %}
+# {% else %}
+# {% set cudnn_cuda_version = cuda_compiler_version %}
+# {% endif %}
```

Comment:
@isuruf I tested it locally, and found that if I remove the cfc file in favor of the migrators (like what you showed in #24), and *then* restore this snippet to set and use `cudnn_cuda_version`, the `cudnn` key comes back again to the CI config files. I don't understand why you considered it ok -- in this case both `cudnn7` and `cudnn8` would be added, duplicating the number of tests to run. Am I missing anything?

Reply:
sorry nvm, I read the content wrong...